packages/Webkul/Velocity/src/Resources/views/shop/home/featured-products.blade.php

```diff
@@ -1,11 +1,12 @@
 @php
     $count = core()->getConfigData('catalog.products.homepage.no_of_featured_product_homepage') ?: 10;
-
+    $guest_review_status = core()->getConfigData('catalog.products.review.guest_review');
     $direction = core()->getCurrentLocale()->direction == 'rtl' ? 'rtl' : 'ltr';
 @endphp
```

Comment:
Move all this logic to the controller. This blade file should be clean and should only have the required variables.